kiwi/schema/kiwi.rnc

```diff
@@ -187,9 +191,11 @@ div {
 div {
     k.archive.name.attribute = k.name.attribute
     k.archive.bootinclude.attribute = k.bootinclude.attribute
+    k.archive.target_dir.attribute = k.target_dir.attribute
     k.archive.attlist =
         k.archive.name.attribute &
-        k.archive.bootinclude.attribute?
+        k.archive.bootinclude.attribute? &
+	k.archive.target_dir.attribute?
```

Comment:
A few issues here:

1. There is no need to introduce a new attribute specifier (k.target_dir.attribute). The content for target_dir is a path and for this there is already ```k.path.attribute```
2. small Indentation issue

So the following should be enough:

```
k.archive.target_dir.attribute = k.path.attribute

k.archive.attlist =
        k.archive.name.attribute &
        k.archive.bootinclude.attribute? &
        k.archive.target_dir.attribute?
```